benchmark.py

```diff
@@ -43,6 +39,19 @@ def __init__(self, db_tpl, db_type):
         self.read = -1
         self.combined = -1
 
+        self.available = self.load_module(db_type if db_module is None else db_module)
```

Comment:
this breaks benchmarks which don't require imports (like DummyPickleBenchmark). better don't fall back to db_type and always pass db_module explicitly.

Reply:
Missed the dummy ones. Changed the code to explicitly set the module name and try to load it if set